---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    EntandoResourceKind: "EntandoApp"
    EntandoResourceNamespace: "my-namespace"
    EntandoApp: "my-app"
  name: "my-app-40207928"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "entando.org/v1"
    kind: "EntandoApp"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "abd9c995-bb5e-4b5e-99e3-a9b8a5ba772b"
action: "FAILED"
count: 1
firstTimestamp: "2021-06-21T16:38:10Z"
involvedObject:
  apiVersion: "entando.org/v1"
  kind: "EntandoApp"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "3471190"
  uid: "abd9c995-bb5e-4b5e-99e3-a9b8a5ba772b"
lastTimestamp: "2021-06-21T16:38:10Z"
message: "The deployment of EntandoApp my-namespace/my-app failed due to Test Exception.\
  \ Fix the root cause and then trigger a redeployment by adding the annotation 'entando.org/processing-instruction:\
  \ force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "entando-k8s-entando-k8s-app-con-hh6lg-22-from-build-pack--4n57c"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-app-controller"
type: "Error"
